doc/release/platforms/macosx.rst

```diff
@@ -17,6 +17,10 @@ following command::
 
     brew install chapel
 
+When using a Homebrew install of Chapel, the ``CHPL_HOME`` directory is
```

Comment:
While we're here, maybe we could mention that one can build Chapel from the head of the repository with `brew install chapel --HEAD` to get the latest and greatest features.